src/main/java/com/minecolonies/coremod/achievements/MineColoniesAchievementPage.java

```diff
@@ -10,17 +9,177 @@
+            final MineColoniesAchievement achievementBuild25Huts,
+            final MineColoniesAchievement achievementBuild100Huts,
+            final MineColoniesAchievement achievementBuild500Huts,
+            final MineColoniesAchievement achievementBuild1000Huts,
+            final MineColoniesAchievement achievementCatchOneFish,
+            final MineColoniesAchievement achievementCatch25Fish,
+            final MineColoniesAchievement achievementCatch100Fish,
+            final MineColoniesAchievement achievementCatch500Fish,
+            final MineColoniesAchievement achievementCatch1000Fish,
+            final MineColoniesAchievement achievementHarvestOneCarrot,
+            final MineColoniesAchievement achievementHarvest25Carrots,
+            final MineColoniesAchievement achievementHarvest100Carrots,
+            final MineColoniesAchievement achievementHarvest500Carrots,
+            final MineColoniesAchievement achievementHarvest1000Carrots,
+            final MineColoniesAchievement achievementHarvestOnePotato,
+            final MineColoniesAchievement achievementHarvest25Potatoes,
+            final MineColoniesAchievement achievementHarvest100Potatoes,
+            final MineColoniesAchievement achievementHarvest500Potatoes,
+            final MineColoniesAchievement achievementHarvest1000Potatoes,
+            final MineColoniesAchievement achievementHarvestOneWheat,
+            final MineColoniesAchievement achievementHarvest25Wheat,
+            final MineColoniesAchievement achievementHarvest100Wheat,
+            final MineColoniesAchievement achievementHarvest500Wheat,
+            final MineColoniesAchievement achievementHarvest1000Wheat,
+            final MineColoniesAchievement achievementFellOneTree,
+            final MineColoniesAchievement achievementFell25Trees,
+            final MineColoniesAchievement achievementFell100Trees,
+            final MineColoniesAchievement achievementFell500Trees,
+            final MineColoniesAchievement achievementFell1000Trees,
+            final MineColoniesAchievement achievementPlantOneSapling,
+            final MineColoniesAchievement achievementPlant25Saplings,
+            final MineColoniesAchievement achievementPlant100Saplings,
+            final MineColoniesAchievement achievementPlant500Saplings,
+            final MineColoniesAchievement achievementPlant1000Saplings,
+            final MineColoniesAchievement achievementMinerDeathLava,
+            final MineColoniesAchievement achievementMinerDeathFall,
+            final MineColoniesAchievement achievementLumberjackDeathTree,
+            final MineColoniesAchievement achievementFisherDeathGuardian,
+            final MineColoniesAchievement achievementGuardDeathEnderman,
+            final MineColoniesAchievement achievementPlayerDeathGuard,
+            final Advancement... achievements)
     {
-        super(name, achievements);
+        //todo will have to call super
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)